nessie/src/main/java/org/apache/iceberg/nessie/UpdateableReference.java

```diff
@@ -62,7 +62,7 @@ public Reference getReference() {
 
   public void checkMutable() {
     Preconditions.checkArgument(
-        mutable, "You can only mutate tables when using a branch without a hash or timestamp.");
+        mutable, "You can only mutate contents when using a branch without a hash or timestamp.");
```

Comment:
"contents" is not a well-known term in the database world. I'd prefer "tables/views" here - not sure if this is used for namespaces as well.

Reply:
updated